docs/installation.rst

```diff
@@ -71,19 +71,21 @@ If you are on Fedora, install the build dependencies with this command:
 
 .. code-block:: bash
 
-   sudo dnf install -y gcc make cmake bzip2-devel expat-devel file-devel glib2-devel libcurl-devel libmodulemd-devel libxml2-devel python3-devel python3-gobject python3-libmodulemd rpm-devel openssl-devel sqlite-devel xz-devel zchunk-devel zlib-devel
+   sudo dnf install -y gcc make cairo-devel cairo-gobject-devel cmake bzip2-devel expat-devel file-devel glib2-devel gobject-introspection-devel libcurl-devel libmodulemd-devel libxml2-devel openssl-devel python3-devel rpm-devel sqlite-devel xz-devel zchunk-devel zlib-devel
 
 If on CentOS or RHEL, use this commands:
 
-Ensure you have enabled ``epel`` repository
+Ensure you have enabled ``epel`` and ``powertools`` repositories
```

Comment:
I think "powertools" is called something else in EL8

Reply:
you are right, it is ``codeready`` - updated